src/libraries/token/StandardToken.sol

```diff
@@ -0,0 +1,57 @@
+pragma solidity ^0.4.11;
+
+import 'ROOT/libraries/token/BasicToken.sol';
+import 'ROOT/libraries/token/ERC20.sol';
+
+/**
+ * @title Standard ERC20 token
+ *
+ * @dev Implementation of the basic standard token.
+ * @dev https://github.com/ethereum/EIPs/issues/20
+ * @dev Based on code by FirstBlood: https://github.com/Firstbloodio/token/blob/master/smart_contract/FirstBloodToken.sol
+ */
+contract StandardToken is ERC20, BasicToken {
+	using SafeMath for uint256;
+
+	mapping (address => mapping (address => uint256)) allowed;
+
+	/**
+	* @dev Transfer tokens from one address to another
+	* @param _from address The address which you want to send tokens from
+	* @param _to address The address which you want to transfer to
+	* @param _value uint256 the amout of tokens to be transfered
+	*/
+	function transferFrom(address _from, address _to, uint256 _value) returns (bool) {
+		var _allowance = allowed[_from][msg.sender];
+
+		// Check is not needed because sub(_allowance, _value) will already throw if this condition is not met
+		// require (_value <= _allowance);
+
+		balances[_to] = balances[_to].add(_value);
+		balances[_from] = balances[_from].sub(_value);
+		allowed[_from][msg.sender] = _allowance.sub(_value);
+		Transfer(_from, _to, _value);
+		return true;
+	}
+
+	/**
+	* @dev Aprove the passed address to spend the specified amount of tokens on behalf of msg.sender.
+	* @param _spender The address which will spend the funds.
+	* @param _value The amount of tokens to be spent.
+	*/
+	function approve(address _spender, uint256 _value) returns (bool) {
+		allowed[msg.sender][_spender] = _value;
+		Approval(msg.sender, _spender, _value);
+		return true;
+	}
+
+	/**
+	* @dev Function to check the amount of tokens that an owner allowed to a spender.
+	* @param _owner address The address which owns the funds.
+	* @param _spender address The address which will spend the funds.
+	* @return A uint256 specifing the amount of tokens still avaible for the spender.
+	*/
+	function allowance(address _owner, address _spender) constant returns (uint256 remaining) {
```

Comment:
Hmm, I didn't even realize that you could name output parameters (this code is cribbed from OpenZeppelin).  I actually like it because it allows me to describe (in code) what is being returned.  In general, when I can describe something in code instead of a comment I find that valuable.  I also suspect that this name _may_ make it into exported interface definitions, which could be valuable.

That being said, I don't feel particularly strongly since, as you hinted at, the variable will never be used so it would be easy to become out of date (just like comments).